ui/src/onboarding/components/configureStep/Scraper.tsx

```diff
@@ -0,0 +1,106 @@
+// Libraries
+import React, {PureComponent, ChangeEvent} from 'react'
+
+// Components
+import {
+  Form,
+  Input,
+  Columns,
+  Grid,
+  ComponentSize,
+  MultipleInput,
+  Dropdown,
+  InputType,
+} from 'src/clockface'
+
+interface Props {
+  bucket: string
+  dropdownBuckets: JSX.Element[]
+  onIntervalHandle: (value: string) => void
+  onDropdownHandle: (value: string) => void
```

Comment:
onXHandle sounds like a strange combination of naming conventions to me, would onChooseInterval or onSelectInterval be better?

Reply:
Changed it!